requirements/requirements.txt

```diff
@@ -106,7 +106,6 @@ constantly==15.1.0        # via twisted
 cryptography==2.1.4       # via adal, azure-cosmosdb-table, azure-storage-common, pyopenssl, requests, secretstorage
 daphne==1.3.0
 decorator==4.2.1
-debtcollector==1.15.0     # via oslo.config, oslo.utils, python-designateclient, python-keystoneclient, python-neutronclient
```

Comment:
It is not clear to me why this dependency is being removed, and I would like to know more!